src/main/java/entityLayer/CommonTimeBlockManager.java

```diff
@@ -1,4 +1,20 @@
 package entityLayer;
 
-public class CommonTimeBlockManager {
+import java.util.List;
+
+public class CommonTimeBlockManager implements TimeBlockManager {
+    @Override
+    public List<TimeBlock> getTimeBlocks() {
+        return null;
+    }
+
+    @Override
+    public void addTimeBlock(TimeBlock timeBlock) {
+
+    }
+
+    @Override
+    public boolean removeTimeBlock(TimeBlock timeBlock) {
+        return false;
+    }
 }
```

Comment:
we talked about a method for checking to make sure that a time block lies within the users availability, not sure if that will still be useful or not

Reply:
The TimeBlocks have methods to see whether they are contained within/overlap with other TimeBlocks, so I suspected that we'd simply be using those instead. As I last remember from how we discussed it, the availability is a manager with only 7 timeblocks (one for each day of the week), so doing this shouldn't be too difficult, I don't think.